proto/osmosis/protorev/v1beta1/protorev.proto

```diff
@@ -87,6 +87,8 @@ message PoolWeights {
   // The weight of a concentrated pool
   uint64 concentrated_weight = 3
       [ (gogoproto.moretags) = "yaml:\"concentrated_weight\"" ];
+  // The weight of a cosmwasm pool
+  uint64 cosmwasm_weight = 4 [ (gogoproto.moretags) = "yaml:\"cw_weight\"" ];
```

Comment:
Can we change this yaml to match the proto name `cosmwasm_weight` (we don't abbreviate any other protos/fields so prob shouldn't start now)?

Reply:
good catch